sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/LinuxVMGuestPatchAutomaticByPlatformRebootSetting.cs

```diff
@@ -12,11 +12,13 @@ namespace Microsoft.Azure.Management.Compute.Models
 {
 
     /// <summary>
-    /// Defines values for ResultStatus.
+    /// Defines values for LinuxVMGuestPatchAutomaticByPlatformRebootSetting.
```

Comment:
review

Reply:
"ResultStatus" is an object from 54.1.0-preview release, which is being over written